powerline_shell/__init__.py

```diff
@@ -134,6 +134,7 @@ def find_config():
     for location in [
         "powerline-shell.json",
         "~/.powerline-shell.json",
+        os.path.join(os.environ.get("XDG_CONFIG_HOME", ""), "powerline-shell", "config.json"),
```

Comment:
Perhaps it would be better to set a default:
```
os.path.join(os.environ.get("XDG_CONFIG_HOME", "~/.config"), "powerline-shell", "config.json"),
```
since many systems don't set the `XDG_CONFIG_HOME` environment variable.